english.html

```diff
@@ -246,7 +242,7 @@ <h1>Advantages of Qint</h1>
           <div class="content revealBottom" style="text-align: center; margin-top: 1em">
             <p>Provide your organization and customers the innovation they deserve</p>
             <a href="https://app.q-int.com/#/register" onclick="emitEvent('loginBottom')">
-              <button class="btn callToAction">Lets begin</button>
+              <button class="btn callToAction">Let's begin</button>
```

Comment:
Food for thought: When is it appropriate to use just `'`, and when to use `&apos;`?

Reply:
I am guessing but, using the code version is better for more browsers?